### DISCUSSION TITLE
Redis cache username

### DISCUSSION THREAD

**Discussion by superbeppe77** [OpenIDC/mod_auth_openidc#932] [Q&A]
Hi,
when I use redis as cache with
OIDCRedisCacheServer
I can set a password (OIDCRedisCachePassword) but not username (from redis 6 acl are possible)
In commercial version is it possible specify also redis username?
Best Regards

**Comment by zandbelt** [comment]
I think setting `OIDCRedisCachePassword "myuser mypassword"` should work; if you can confirm I'll add it to the docs

**Reply by superbeppe77** [reply]
Hi,
I tried that with mod_auth_openidc-2.4.10
but it doesn't work.

Apache-Error: [file "src/cache/redis.c"] [line 238] [level 3] oidc_cache_redis_connect: Redis AUTH command (dev-pbp-redis:6379) failed: '' [WRONGPASS invalid username-password pair or user is disabled.]

The user is active and it works with AUTH in redis-cli

**Reply by zandbelt** [reply]
I see, that would require development of a new feature indeed, which is driven by PR contributions or commercial demand

**Comment by zandbelt** [comment]
we had a nearly finished patch for this feature in store already, completed now and applied here:
https://github.com/zmartzone/mod_auth_openidc/commit/63094742d40713bb24901db838446c53df8016f8

**Reply by superbeppe77** [reply]
Great!
I inform the team that build and install the module about that.

Thank you :)
